### PR TITLE
build: automatically cleanup docker container

### PR DESCRIPTION
**Pull request by julianoes**
The argument `--rm` automatically cleans up leftover docker containers
after you exit the docker container.

To cleanup old containers, you can also use:
```
docker system prune
```

### CODE REVIEW

**Comment by julianoes** [comment]
> Is there any case that I might want to not clean up my docker container when I exit.? The only case I can think of perhaps is the first/bash case - say I want to come back and reuse it. Is this likely?

In the case where you re-attach to containers you know how it works and you'll just remove the `--rm`. For the rest of us that just want to build it using docker, I think it's a sensible default.

File: en/contributing/build.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ The main steps are:
    ```
 1. Enter the following command in your host's terminal:
    ```sh
-   docker run -it -v $HOME/<path-to-dronecore-repo>/DroneCore:/home/docker1000/src/DroneCore:rw dronecore/dronecore bash
+   docker run --rm -it -v $HOME/<path-to-dronecore-repo>/DroneCore:/home/docker1000/src/DroneCore:rw dronecore/dronecore bash
    ```
    > **Note** The `-v` flag maps a directory on your host (left side) to a path in
```
Comment (hamishwillee) [inline en/contributing/build.md]: How about we add a note about the --rm flag, as below:

```
> **Note** The `-v` flag maps a directory on your host (left side) to a path in the container (right side). 
> You need to specify the left-side path to the DroneCore repository on your host and the container path must be set as above. 
> The `--rm` automatically cleans up leftover docker containers after you exit the docker container.
```

**Comment by hamishwillee** [comment]
@julianoes OK, sounds very reasonable. How about we add a note explaining --rm too (see comment inline). Then happy to merge.

File: en/contributing/build.md (hunk 1 above)
Comment (julianoes) [reply en/contributing/build.md]: Done 9a1896c

**Comment by hamishwillee** [comment]
Thanks. Merged.!

**Pull request merged**